src/nss/nw_s1_auramenca.nss

```diff
@@ -40,5 +45,6 @@ void main()
             ApplyEffectToObject(DURATION_TYPE_INSTANT, eVis, oTarget);
             ApplyEffectToObject(DURATION_TYPE_TEMPORARY, eLink, oTarget, TurnsToSeconds(nDuration));
         }
+        SetLocalInt(OBJECT_SELF, "doomaura" + ObjectToString(oTarget), 1);
```

Comment:
nice the object to string identifier probably makes it unique to different creatures with the aura